lightly/api/patch.py

```diff
@@ -0,0 +1,58 @@
+import logging
+from typing import Any, Dict, Type
+
+
+def make_swagger_configuration_picklable(
+    configuration_cls: Type,
+) -> None:
```

Comment:
Here you overwrite attributes of the original `Configuration` to create the patched version. Why is this done using a mixin for the `LightlySwaggerApiClient` and `LightlySwaggerRESTClientObject`? Why cannot you use the way to overwrite some attributes for all 3 cases?